integration-test/src/main/java/com/sequenceiq/it/cloudbreak/action/sdx/SdxBackupAction.java

```diff
@@ -29,22 +30,32 @@ public SdxBackupAction(String backupLocation, String backupName) {
     }
 
     @Override
-    public SdxInternalTestDto action(TestContext testContext, SdxInternalTestDto testDto, SdxClient client) throws Exception {
+    public SdxTestDto action(TestContext testContext, SdxTestDto testDto, SdxClient client) throws Exception {
         String sdxName = testDto.getName();
 
-        Log.when(LOGGER, format(" SDX '%s' backup has been started to '%s' as '%s' ", sdxName, backupLocation, backupName));
+        sleep(2, sdxName);
+
+        Log.when(LOGGER, format(" SDX '%s' backup has been started to '%s' by name '%s'... ", sdxName, backupLocation, backupName));
         Log.whenJson(LOGGER, " SDX backup request: ", testDto.getRequest());
-        LOGGER.info(format(" SDX '%s' backup has been started to '%s' as '%s'... ", sdxName, backupLocation, backupName));
+        LOGGER.info(format(" SDX '%s' backup has been started to '%s' by name '%s'... ", sdxName, backupLocation, backupName));
         SdxBackupResponse sdxBackupResponse = client.getDefaultClient()
                 .sdxBackupEndpoint()
                 .backupDatalakeByName(sdxName, backupLocation, backupName);
         testDto.setFlow("SDX backup", sdxBackupResponse.getFlowIdentifier());
-
         SdxClusterDetailResponse detailedResponse = client.getDefaultClient()
                 .sdxEndpoint()
                 .getDetail(sdxName, Collections.emptySet());
         testDto.setResponse(detailedResponse);
-        Log.whenJson(LOGGER, " SDX backup response: ", client.getDefaultClient().sdxEndpoint().get(sdxName));
+        Log.whenJson(LOGGER, " SDX response after backup: ", client.getDefaultClient().sdxEndpoint().get(sdxName));
         return testDto;
     }
+
+    private void sleep(long sleepMinutes, String sdxName) {
+        try {
+            TimeUnit.MINUTES.sleep(sleepMinutes);
+        } catch (InterruptedException ignored) {
+            LOGGER.warn("Waiting for Cloudera Manager services have been synchronize then all up and running at Sdx '{}'" +
+                    " has been interrupted at resource {}, because of: {}", sdxName, ignored.getMessage(), ignored);
```

Comment:
I am not sure I understand this error message:  "Waiting for Cloudera Manager services have been synchronize then all up and running at Sdx [sdxname]  has been interrupted at resource [error_message], because of: [stacktrace]". This would be better: "Waiting for CM services to be synchronized has been interrupted, cause: [stacktrace]."

Reply:
Thank you, the proposed one sounds better!